src/gleescript.gleam

```diff
@@ -51,17 +51,34 @@ pub fn run() -> snag.Result(Nil) {
         |> snag_inspect_error
         |> snag.context("Failed to read " <> f),
       )
+
       #(name, content)
     }),
   )
 
+  use _ <- result.try(case simplifile.verify_is_directory(config.target_dir) {
+    Ok(False) -> {
+      simplifile.create_directory_all(config.target_dir)
+      |> snag_inspect_error
+      |> snag.context("Failed to create " <> config.target_dir <> " directory")
+    }
+    Error(e) ->
+      Error(e)
+      |> snag_inspect_error
+      |> snag.context("Failed to read " <> config.target_dir <> " directory")
+    Ok(True) -> Ok(Nil)
+  })
```

Comment:
We can call `create_directory_all` without calling `verify_is_directory` first here.

Reply:
noted